src/org/ensembl/healthcheck/testcase/compara/AbstractComparaTestCase.java

```diff
@@ -235,22 +235,33 @@ public DatabaseRegistryEntry getLastComparaReleaseDbre(DatabaseRegistryEntry cur
 	protected final static Pattern EC_DB = Pattern.compile("^([^_]+_)?ensembl_compara_[0-9]+");
 
 	String getComparaDivisionName(DatabaseRegistryEntry compara_dbre) {
+		Connection compara_con = compara_dbre.getConnection();
+		String sql_division = "SELECT meta_value FROM meta WHERE meta_key = 'division'";
+		String division = DBUtils.getRowColumnValue(compara_con, sql_division);
+		if (division == null || division.isEmpty()) {
 		Matcher m = EGC_DB.matcher(compara_dbre.getName());
 		if (m.matches()) {
 			if (m.groupCount() > 1) {
-				return m.group(2);
+				division = m.group(2);
 			} else {
-				return m.group(1);
+				division = m.group(1);
 			}
 		} else {
 			m = EC_DB.matcher(compara_dbre.getName());
 			if (m.matches()) {
-				return "ensembl";
+				division = "ensembl";
 			} else {
 				ReportManager.problem(this, compara_dbre.getConnection(), "Cannot find the division name of this database: '" + compara_dbre.getName() + "'");
 				return null;
 			}
 		}
+		}
+		// "ensembl" division has been renamed to "vertebrates"
```

Comment:
The EGC_DB regex only works for EG, which won't generate a division named "ensembl". So if you set the division to "vertebrates" up there, there is no need to do this transformation here

Reply:
This would mean you don't need to use the variable `division` in the pattern-matching part. Just return the string immediately (like it was before)